package.json

```diff
@@ -18,7 +18,9 @@
     "build": "lerna run build",
     "clean": "lerna run clean",
     "build:tsc": "tsc -b packages/**/tsconfig.build.json",
-    "lint": "eslint . --fix --quiet",
+    "lint": "lerna run lint:check",
+    "lint:ci": "yarn format:check && yarn lint:check && yarn packageJsonLint",
+    "lint:fix": "eslint . --fix --quiet",
```

Comment:
If you accept my other suggestion, then no need for this one, since `lerna run lint` would already take care of this.

```suggestion
```